docs/source/publications.rst

```diff
@@ -23,6 +25,10 @@ Coupling
 Analysis
 --------
 
+- Rémi Delaporte-Mathurin et al., *3D Effects on Hydrogen Transport in ITER-like Monoblocks* Nuclear Fusion, (2023), https://doi.org/10.1088/1741-4326/ad1019.
+
+- Rémi Delaporte-Mathurin, “Hydrogen Transport in Tokamaks: Estimation of the ITER Divertor Tritium Inventory and Influence of Helium Exposure” (These de doctorat, Paris 13, 2022), https://www.theses.fr/2022PA131054.
```

Comment:
```suggestion
- Rémi Delaporte-Mathurin, *Hydrogen Transport in Tokamaks: Estimation of the ITER Divertor Tritium Inventory and Influence of Helium Exposure* (These de doctorat, Université Sorbonne Paris Nord, 2022), https://www.theses.fr/2022PA131054.
```

Although maybe its not in italics for a thesis? don't know

Reply:
Let's be consistent